controls/radchartview/series/cartesianchart-series/bar-series/rangebarseries.md

```diff
@@ -12,12 +12,17 @@ position: 1
 
 This series is visualized on the screen as separate rectangles representing each of the __DataPoints__. The main difference of the BarSeries is that you are allowed to define range with each data point.      
 
+* [Declaratively Defined Series](#declaratively-defined-series)
+* [Properties](#properties)
+* [Data Binding](#data-binding)
+* [Styling the Series](#styling-the-series)
+
 ## Declaratively defined series
 
 You can use the following definition to display a simple RangeBarSeries

```

Comment:
You can replace this sentence with the following one.

"_You can use the definition from __Example 1__ to display an RangeBarSeries._"